examples/child_window.rs

```diff
@@ -57,24 +57,27 @@ fn main() -> Result<(), impl std::error::Error> {
                     ..
                 } => {
                     let parent_window = self.windows.get(&self.parent_window_id.unwrap()).unwrap();
-                    let child_window = spawn_child_window(parent_window, event_loop);
+                    let child_window = spawn_child_window(parent_window.as_ref(), event_loop);
```

Comment:
How about `&*parent_window`?

Reply:
Nah, I guess you have to do exactly the same as we do with the `ApplicationHandler` and all the generic impls over there, I don't mind them, but I'll leave it to follow-up, since there's a chance that none of these `as_ref` will be needed once we figure out how we actually store the window for the end user.